packages/console/src/pages/AuditLogDetails/index.tsx

```diff
@@ -22,19 +23,25 @@ import EventIcon from './components/EventIcon';
 import * as styles from './index.module.scss';
 
 const AuditLogDetails = () => {
-  const { logId } = useParams();
+  const { id: userId, logId } = useParams();
+  const { pathname } = useLocation();
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const { data, error } = useSWR<LogDTO, RequestError>(logId && `/api/logs/${logId}`);
+  const { data: userData } = useSWR<User, RequestError>(userId && `/api/users/${userId}`);
+
   const isLoading = !data && !error;
 
+  const backLink =
+    pathname.startsWith('/users') && userId ? `/users/${userId}/logs` : '/audit-logs';
```

Comment:
check `userId` is enough? be careful on `pathname`, this may vary in production env.